proto/observer/params.proto

```diff
@@ -19,11 +20,21 @@ message CoreParams {
   string zeta_token_contract_address = 8;
   string connector_contract_address = 9;
   string erc20_custody_contract_address = 10;
-  int64 chain_id = 11;
   int64 outbound_tx_schedule_interval = 12;
   int64 outbound_tx_schedule_lookahead = 13;
+  string ballot_threshold = 14 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+  string min_observer_delegation = 15 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+  bool is_supported = 16;
```

Comment:
while you are at it, where is `is_supported` used? This wording kind of bothers me (because if it's not supported why do we have an entry here), maybe something like `is_active` is better?

Reply:
I think it is good to have a bool we can toggle to quickly stop supporting a chain with a group proposal.
I agree about the renaming into `IsActive` https://github.com/zeta-chain/node/issues/1523